src/style.css

```diff
@@ -1,3 +1,86 @@
 body {
   margin: 0;
+  padding: 0;
+  display: block;
+}
+
+.header {
+  box-sizing: border-box;
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background: #fff;
+}
+
+.logo_img {
+  width: 40px;
+  height: 40px;
+}
```

Comment:
Don't use tag names for styling (except `html` and `body`). Style all elements using `.class`. You should give the `img` element a class and style it using that class instead of using the tag name in your selector.